env/dev/figwheel.clj

```diff
@@ -0,0 +1,47 @@
+(ns figwheel
+  (:require [clojure.string :as s]))
+
+(def all-builds
+  [{:id           :ios
+    :source-paths ["react-native/src" "src" "env/dev"]
+    :compiler     {:output-to     "target/ios/app.js"
+                   :main          "env.ios.main"
+                   :output-dir    "target/ios"
+                   :optimizations :none}
+    :figwheel     true}
+   {:id               :android
+    :source-paths     ["react-native/src" "src" "env/dev"]
+    :compiler         {:output-to     "target/android/app.js"
+                       :main          "env.android.main"
+                       :output-dir    "target/android"
+                       :optimizations :none}
+    :warning-handlers '[status-im.utils.build/warning-handler]
+    :figwheel         true}])
+
+(defn get-test-build [build id]
+  (-> build
+      (update :source-paths
+              (fn [paths] (let [paths-set (set paths)]
+                            (-> paths-set
+                                (disj "env/dev")
+                                (conj "test/cljs")
+                                vec))))
+      (assoc :id id)))
+
+(defn test-id? [id]
+  (s/includes? (name id) "-test"))
+
+(defn get-id [id]
+  (-> id
+      name
+      (s/replace #"-test" "")
+      keyword))
+
+(defn get-build [builds id]
+  (when-let [build (first (filter #(= (:id %) (get-id id)) builds))]
+    (if (test-id? id)
```

Comment:
Is this part still relevant? If not we could simply remove it.

Reply:
Not that I can see, at least in terms of immediate code breakage. It was kept for the case of everyday REPL session use, but as far as I can tell the *-test runs are already broken in develop. 

Maybe we can go ahead and delete the code, and see if there any complaints? I think the chances of that happening are low enough.

Reply:
Please remove it then! :)